subsys/bluetooth/services/hids.c

```diff
@@ -1122,7 +1122,7 @@ int bt_hids_inp_rep_send_userdata(struct bt_hids *hids_obj,
 {
 	struct bt_hids_inp_rep *hids_inp_rep =
 	    &hids_obj->inp_rep_group.reports[rep_index];
-	struct bt_gatt_attr *rep_attr =
+	const struct bt_gatt_attr *rep_attr =
```

Comment:
Could you also apply the same update to other functions that send GATT notifications with HID input reports (to maintain consistency)?